bugbug/db.py

```diff
@@ -157,15 +146,9 @@ def _db_open(path, mode):
             yield store_constructor(f)
     elif compression == "zstd":
         if "w" in mode or "a" in mode:
-            cctx = zstandard.ZstdCompressor()
-            with open(path, mode) as f:
-                with cctx.stream_writer(f) as writer:
-                    yield store_constructor(writer)
+            utils.zstd_compress(path)
         else:
-            dctx = zstandard.ZstdDecompressor()
-            with open(path, mode) as f:
-                with dctx.stream_reader(f) as reader:
-                    yield store_constructor(reader)
+            utils.zstd_decompress(path)
```

Comment:
This is wrong, the old code was not decompressing a file. In this case, keep the old code.